docs/advanced/data_types.mdx

```diff
@@ -93,6 +93,16 @@ app.add("my-page-cfbc134ca6464fc980d0391613959196", "notion")
 app.add("https://www.notion.so/my-page-cfbc134ca6464fc980d0391613959196", "notion")
 ```
 
+### Wikipedia
+
+To add any wikipedia page, use the data_type as `wikipedia`. Eg:
+
+```python
+app.add("Wikipedia Page Title", data_type="wikipedia")
```

Comment:
could you make this work with URLs too?

Reply:
The package I'm using doesn't allow directly taking URLs, It uses the wikipedia search and suggest endpoint to find a page. Refer: https://wikipedia.readthedocs.io/en/latest/code.html

Other way to achieve this would be using BS4, but that would become complicated with tabular data.

Reply:
ok, I was just wondering.